docs/_pages/download.cn.md

```diff
@@ -7,29 +7,29 @@ last_modified_at: 2019-12-30T15:59:57-04:00
 ---
 
 ## Release 0.8.0
-* Source Release : [Apache Hudi 0.8.0 Source Release](https://downloads.apache.org/hudi/0.8.0/hudi-0.8.0.src.tgz) ([asc](https://downloads.apache.org/hudi/0.8.0/hudi-0.8.0.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.8.0/hudi-0.8.0.src.tgz.sha512))
+* Source Release : [Apache Hudi 0.8.0 Source Release](https://www.apache.org/dyn/closer.lua/hudi/0.8.0/hudi-0.8.0.src.tgz) ([asc](https://downloads.apache.org/hudi/0.8.0/hudi-0.8.0.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.8.0/hudi-0.8.0.src.tgz.sha512))
```

Comment:
https://www.apache.org/dyn/closer.lua/hudi/ redirects me to https://apache.osuosl.org/hudi/ fyi.

Reply:
probably it chooses a mirror location closer to your geo location.

Reply:
probably. www.apache.org/dyn/closer.lua was also mentioned in the instruction email sent by the owner of announce@apache.org, so I think this should be the right one to put on the site.